opsdroid/cli/config.py

```diff
@@ -49,3 +49,10 @@ def gen(ctx):
 def edit(ctx):
     """Print out the example config."""
     edit_files(ctx, None, "config")
+
+
+@config.command()
+@click.pass_context
+def validate(ctx):
```

Comment:
This is a really nice touch, I hadn't thought of this.

However I think I would prefer the word `lint` to `validate`. Thoughts?

Reply:
I don't really mind changing this to lint, the only reason why I choose validate was to combine with the config command. If you prefer the word lint I'm happy to change it 😄

Reply:
Yeah would you mind changing it? I think `lint` is used in quite a few cli tools and would feel more natural.